pkg/utils/utils.go

```diff
@@ -153,3 +157,16 @@ func findChartByVersion(chartVersions []*helmRepo.ChartVersion, version string)
 	}
 	return nil
 }
+
+// isValidUrl tests a string to determine if it is a well-structured url or not.
+func isValidURL(text string) bool {
+	_, err := url.ParseRequestURI(text)
+	if err != nil {
+		return false
+	}
+	u, err := url.Parse(text)
```

Comment:
Is it really needed to parse it twice? Here you can find how both `Parse` and `ParseRequestURI` basically share the same parsing method:
 - https://golang.org/src/net/url/url.go?s=14096:14145#L485
 - https://golang.org/src/net/url/url.go?s=13410:13449#L464

If it does matter, please add a comment.

Reply:
Probably not needed. I took the function from somewhere else and use it as it was.